### PR TITLE
[MINI 4858] Improve branch code coverage 

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
This PR aims to improve branch code coverage. Please refer to the below comparison:

### Old branch code coverage (69%)
old-cov

### New branch code coverage (70%)
Screen Shot 2022-12-12 at 16 56 53

## Links
- MINI 4858

# Checklist
- [x] I have read the [contributing guidelines](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md#sdk-development-learning-path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by shohiebsense — APPROVED** [review]
those 🟢 💚 🟢 feelsgoodman

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/util/MiniAppDatabaseEncryptionUtil.kt [hunk 1]
```diff
@@ -155,7 +155,7 @@ internal object MiniAppDatabaseEncryptionUtil {
 
         val holder = getPasscodeHolder(context)
 
-        if (holder == null || (holder.salt == null || holder.iv == null || holder.encryptedPasscode == null)) {
+        if (holder == null) {
```
Comment (shohiebsense) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/util/MiniAppDatabaseEncryptionUtil.kt:158]: u sure about this?

Comment (khairul-alam-licon) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/util/MiniAppDatabaseEncryptionUtil.kt:158]: Yeah, basically we should check only `holder == null` and we can remove the other conditions as the properties are non-nullable e.g. [EncryptedPasscodeDataHolder](https://github.com/rakutentech/android-miniapp/blob/master/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/util/MiniAppDatabaseEncryptionUtil.kt#L35~L38). Removing those redundant conditions would improve branch coverage in this part. 
However, we need to add condition in future accordingly if we change any property.

**Review by munir-rakuten — APPROVED** [review]
Looks good to me 👍

**Pull request merged**